epochStart/bootstrap/epochStartMetaBlockProcessor.go

```diff
@@ -191,6 +192,26 @@ func (e *epochStartMetaBlockProcessor) GetEpochStartMetaBlock(ctx context.Contex
 	}
 }
 
+func (e *epochStartMetaBlockProcessor) getMostReceivedMetaBlock() (*block.MetaBlock, error) {
+	e.mutReceivedMetaBlocks.RLock()
+	defer e.mutReceivedMetaBlocks.RUnlock()
+
+	const hashNotAvailable = "N/A"
+	mostReceivedHash := hashNotAvailable
+	maxLength := minNumOfPeersToConsiderBlockValid - 1
+	for hash, entry := range e.mapMetaBlocksFromPeers {
+		if len(entry) > maxLength {
```

Comment:
this is minLength

Reply:
no, because it refers to the length of the peers slice for a metablock

Reply:
and we are interested to get the hash for the longest slice -> the most received

Reply:
maxLength should be updated.